Fix flakyiess of canary/raw disk ci

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

I have noticed multiple flakiness of CI canary/raw disk

```yaml
+ tests/scripts/github-action-helper.sh create_operator_toolbox
++ head -1
++ awk '/14G/ {print $1}'
++ sudo lsblk --paths
+ : /dev/sdb
+ NETWORK_ERROR='connection reset by peer'
+ SERVICE_UNAVAILABLE_ERROR='Service Unavailable'
+ INTERNAL_ERROR=INTERNAL_ERROR
+ INTERNAL_SERVER_ERROR='500 Internal Server Error'
+ FUNCTION=create_operator_toolbox
+ shift
+ create_operator_toolbox
+ cd deploy/examples
+ kubectl create -f toolbox-operator-image.yaml
deployment.apps/rook-ceph-tools-operator-image created
+ kubectl -n rook-ceph wait --for=condition=ready pod -l app=rook-ceph-tools-operator-image --timeout=[18](https://github.com/rook/rook/actions/runs/5874877356/job/15930378443?pr=12679#step:12:19)0s
error: timed out waiting for the condition on pods/rook-ceph-tools-operator-image-6d58bc6bbf-65ltj
Error: Process completed with exit code 1.
```

**Deviation from expected behavior:**

**Expected behavior:**
CI canary/ raw disk should always pass

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary

**Logs to submit**:

* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

  To get logs, use `kubectl -n <namespace> logs <pod name>`
  When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
  Read [GitHub documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Cluster Status to submit**:

* Output of krew commands, if necessary

  To get the health of the cluster, use `kubectl rook-ceph health`
  To get the status of the cluster, use `kubectl rook-ceph ceph status`
  For more details, see the [Rook Krew Plugin](https://rook.io/docs/rook/latest-release/Troubleshooting/krew-plugin)

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/latest-release/Troubleshooting/ceph-toolbox/#interactive-toolbox)):

--- a/.github/workflows/canary-integration-test.yml
+++ b/.github/workflows/canary-integration-test.yml
@@ -292,6 +292,7 @@ jobs:
         run: |
           export ALLOW_LOOP_DEVICES=true
           tests/scripts/github-action-helper.sh deploy_cluster loop
+          tests/scripts/github-action-helper.sh create_operator_toolbox
 
       - name: wait for prepare pod
         run: tests/scripts/github-action-helper.sh wait_for_prepare_pod
@@ -301,7 +302,6 @@ jobs:
 
       - name: test toolbox-operator-image pod
         run: |
-          tests/scripts/github-action-helper.sh create_operator_toolbox
           # waiting for toolbox operator image pod to get ready
           kubectl -n rook-ceph wait --for=condition=ready pod -l app=rook-ceph-tools-operator-image  --timeout=180s
 

